packages/backend/src/functions/storage.ts

```diff
@@ -105,22 +105,49 @@ export const checkIfObjectExist = functions.https.onCall(
 /**
  * Generate a new AWS S3 pre signed url to upload/download an object (GET).
  */
-export const generateGetObjectPreSignedUrl = functions.https.onCall(async (data: any): Promise<any> => {
-    if (!data.bucketName || !data.objectKey) logMsg(GENERIC_ERRORS.GENERR_MISSING_INPUT, MsgType.ERROR)
+export const generateGetObjectPreSignedUrl = functions.https.onCall(
+    async (data: any, context: functions.https.CallableContext): Promise<any> => {
+        // requires auth
+        if (!context.auth) logMsg(GENERIC_ERRORS.GENERR_NO_AUTH_USER_FOUND, MsgType.ERROR)
+
+        if (!data.bucketName || !data.objectKey) logMsg(GENERIC_ERRORS.GENERR_MISSING_INPUT, MsgType.ERROR)
+
```

Comment:
Missing check for process.env.CONFIG_CEREMONY_BUCKET_POSTFIX correct configuration.